velox/functions/sparksql/Hash.cpp

```diff
@@ -390,11 +403,18 @@ std::shared_ptr<exec::VectorFunction> makeHash(
 }
 
 std::vector<std::shared_ptr<exec::FunctionSignature>> xxhash64Signatures() {
-  return {exec::FunctionSignatureBuilder()
-              .returnType("integer")
-              .argumentType("any")
-              .variableArity()
-              .build()};
+  return {
+      exec::FunctionSignatureBuilder()
+          .returnType("integer")
+          .argumentType("any")
+          .variableArity()
+          .build(),
+      exec::FunctionSignatureBuilder()
```

Comment:
The function can be
xxhash(val) and xxhash(val, seed)
`seed` could be the second argument, it complies with spark constructor `XxHash64` and argument with default value maybe the later in the common experience

Reply:
The last argument is marked as VariableArity. We cannot put the `seed` argument after that. https://facebookincubator.github.io/velox/develop/scalar-functions.html
```
The concat function takes an arbitrary number of varchar inputs and returns a varchar. FunctionSignatureBuilder allows specifying that the last augment may appear zero or more times by calling variableArity() method.
```